### ISSUE TITLE
model question

### ISSUE THREAD

**Issue by 1997tzd** [KL4805/CrossTReS#1]
May I ask you some questions?In this model, the last out (NY) is 16 * 458, not 460 obtained by 20 * 23, so how to restore the image in the paper?

**Comment by KL4805** [comment]
It seems that you are running dropoff prediction. As I can remember, the NY taxi dropoff data has two regions without any dropoffs logged. They are not involved in counting error. Therefore, you observe 16*458. 

To disable it, you can simply disable the `mask` (or `spatial_mask`), e.g. [here](https://github.com/KL4805/CrossTReS/blob/main/src/run_crosstres.py#L434). `spatial_mask` is intended to mask out regions without any data.

**Comment by 1997tzd** [comment]
thank you



---- Replied Message ----
| From | Yilun ***@***.***> |
| Date | 02/21/2023 11:58 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [KL4805/CrossTReS] model question (Issue #1) |

It seems that you are running dropoff prediction. As I can remember, the NY taxi dropoff data has two regions without any dropoffs logged. They are not involved in counting error. Therefore, you observe 16*458.

To disable it, you can simply disable the mask (or spatial_mask), e.g. here. spatial_mask is intended to mask out regions without any data.

**Comment by 1997tzd** [comment]
I'm sorry to bother you again. When running your data, I found that the traffic data across cities (NY, CHI) is the same length (8784), and the training set, verification set and test set are divided in the same proportion, so how do you reflect the data characteristics of less samples in migration learning? This is my first contact in this direction. I don't quite understand many places. Thank you for your understanding.

**Comment by KL4805** [comment]
There is a argument in the input, `--data_amount`, which defines the number of target data (in days). You can set it to 3, 7, 10, or whatever you want. By default, it is set to 0, which means full data. 

Whatever you set `--data_amount`, the validation and target data are the same.

**Comment by 1997tzd** [comment]
thank you



---- Replied Message ----
| From | Yilun ***@***.***> |
| Date | 02/22/2023 21:10 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [KL4805/CrossTReS] model question (Issue #1) |

There is a argument in the input, --data_amount, which defines the number of target data (in days). You can set it to 3, 7, 10, or whatever you want. By default, it is set to 0, which means full data.

Whatever you set --data_amount, the validation and target data are the same.
